cfgov/data_research/mortgage_utilities/fips_meta.py

```diff
@@ -330,6 +331,51 @@ def load_fips_meta(counties=True):
     load_constants()
 
 
+@transaction.atomic
+def load_states():
+    """
+    Load state objects from csv data
+    """
+    State.objects.all().delete()
+    states = {}
+
+    with open("{}/states.csv".format(FIPS_DATA_PATH), "r") as f:
+        reader = csv.DictReader(f)
+        for row in list(reader):
+            states[row["abbr"]] = {
+                "fips": row["fips"],
+                "abbr": row["abbr"],
+                "name": row["name"],
+                "counties": [],
+                "msas": [],
+                "non_msa_counties": [],
+            }
```

Comment:
```suggestion
        for row in list(reader):
            if row["abbr"] not in NON_STATES:
                states[row["abbr"]] = {
                    "fips": row["fips"],
                    "abbr": row["abbr"],
                    "name": row["name"],
                    "counties": [],
                    "msas": [],
                    "non_msa_counties": [],
                }
```

Suggestion to keep Puerto Rico out of the states list, unless policies have changed.